src/runtime_src/core/common/info_vmr.cpp

```diff
@@ -72,20 +72,30 @@ vmr_info(const xrt_core::device* device)
 }
 
 bool
-is_default_boot(const xrt_core::device* device)
+get_vmr_status(const xrt_core::device* device, const vmr_status_type& status)
```

Comment:
Ah, this is confusing maybe, but prefer passing primitive types by value.  An enumeration is a primitive type in this context.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in

```suggestion
get_vmr_status(const xrt_core::device* device, vmr_status_type status)
```